pkg/proto/query_coord.proto

```diff
@@ -995,6 +997,38 @@ message RunAnalyzerRequest{
     bool with_hash = 7;
 }
 
+message HighlightTask{
+    string field_name = 1;
+    int64 field_id = 2;
+    repeated string target_texts = 3;
+    repeated string target_analyzers = 4;
+
+    repeated string texts = 5;
+    repeated string analyzers = 6;
+}
+
+message GetHighlightRequest{
+    common.MsgBase base = 1;
+    string channel = 2;
+    repeated int64 topks = 3;
+    repeated HighlightTask tasks=4;
```

Comment:
why do we have multiple tasks?

Reply:
Highlight only run for one field and analyzer. But when we use hybrid search, or other highlight type which support multiple field in future, we need fetch highlight for multiple field. Will create more than one task here.